path_config.go

```diff
@@ -92,13 +92,15 @@ func pathConfig(b *azureAuthBackend) *framework.Path {
 					OperationVerb:   "configure",
 					OperationSuffix: "auth",
 				},
+				ForwardPerformanceSecondary: true,
```

Comment:
Is there any reason we are not also setting `ForwardPerformanceStandby`?

Reply:
forgot